e2e/launcher/browsers.js

```diff
@@ -12,13 +12,13 @@ export const browsersForCI = [
 ];
 
 export const browsersForBrowserStack = [
-	'browserstack:chrome',
+	// 'browserstack:chrome',
```

Comment:
Did you observe this to be true? 

The `execution-plan` tests employed spies within Testcafe's request interceptor, which led to a shared memory issue. If the E2E tests for this repo don't use the same kind of spies, then parallel tests may behave properly.

Reply:
Haven't seen these issues yet, but will keep an eye open